client/components/action-card/README.md

```diff
@@ -4,23 +4,23 @@ This is a [`Card` component](../../components/card) that has a call-to-action bu
 
 ## Usage
 
-```es6
+```jsx
 import ActionCard from 'components/action-card';
 
-render() {
+function render() {
 	return (
 		<ActionCard
 			headerText={ 'Header' }
 			mainText={ 'Some text' }
-			classNames={ 'my-classname-1', 'my-classname-2' }
+			classNames={ ( 'my-classname-1', 'my-classname-2' ) }
```

Comment:
I bet this prop should be called `className`. Also, let's wrap the parens with a `classnames()` call, seems like that was the idea here.